TWTValidation/JSON Validator/Processors/TWTJSONObjectValidatorGenerator.m

```diff
@@ -49,6 +54,8 @@ - (instancetype)init
     self = [super init];
     if (self) {
         _objectStack = [[NSMutableArray alloc] init];
+        _referenceNodesToProxyValidators = [[NSMapTable alloc] init];
+        _referentNodesToValidators = [[NSMapTable alloc] init];
```

Comment:
Don’t use alloc/init. Use `strongToStrongObjectsMapTable`. alloc/init might just make a strong-to-strong table, but I haven’t ever seen that documented.